### PR TITLE
[10.0][MIG] project_timeline

### PR DESCRIPTION
**Pull request by carlosdauden**
This module depends of https://github.com/OCA/web/pull/521

@Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
reviewed

**Review by sergio-teruel — APPROVED** [review]
Reviewed

**Comment by pedrobaeza** [comment]
You have to add to oca_dependencies.txt `web` repo for correct CIs.

File: project_timeline/README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Timeline view for projects
+==========================
+
+This module extends the standard tasks actions to add the timeline view to
+them, and also adds the needed fields in the view for handling them.
+
+Usage
+=====
+
+* Go to *Project > Project > Tasks*.
```
Comment (chienandalu) [inline project_timeline/README.rst]: *Project > Search > Tasks*

**Review by chienandalu — APPROVED** [review]
Tested on runbot. Just a minor comment

**Comment by pedrobaeza** [comment]
@carlosdauden please change the README amending the 10.0 migration commit and I'll merge.

**Comment by carlosdauden** [comment]
Changes done

**Pull request merged**